### ISSUE TITLE
Background stays black after changing user desktop directory [BUG]

### ISSUE THREAD

**Issue by wahve3** [neffo/bing-wallpaper-gnome-extension#174]
**Describe the bug**
Since I have changed the Desktop directory for my user account in Ubuntu 22.04, the extension stopped working. I can only see a black background. If I go into the extension settings, however, the thumbnail is shown. It is, however, not being set as a desktop background.

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04
 - Extension Version: 40
 - GNOME Version: 40

**Comment by neffo** [comment]
Does the path name have non-latin characters in it (a-zA-Z0-9)? It's a bug atm, but fix is in the next version.

**Comment by wahve3** [comment]
No, that's not it.

Can I delete all related settings of the extension somehow?

**Comment by neffo** [comment]
Can you please enable debugging in the extension and provide some journal outputs? (With luck it's something fixed in the next version that isn't pushed out yet.)

**Comment by neffo** [comment]
Is this issue still occurring?

**Comment by wahve3** [comment]
Yes, it is. As soon as I have the extension installed, the background is completely black, so I always uninstall it for now. I have enabled debugging in the extension settings, but where can I find the logs?
